CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing Guidelines
+This repo contains alpha release of some of the foundational aspects of V2 of Kubernetes Federation. All the contribution is done via GitHub. Please also read the contributors guide: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
+
+### Sign the CLA
```

Comment:
You started off with just 1 `#` for the heading size and then went to 3. I would use only 2 `#` for the heading size for all these sub-sections.